packages/components/src/base-field/test/index.js

```diff
@@ -16,30 +16,30 @@ const TestField = ( props ) => {
 describe( 'base field', () => {
 	let base;
 
-	beforeEach( () => {
-		base = render( <TestField /> ).container;
-	} );
-
 	it( 'should render correctly', () => {
+		base = render( <TestField /> ).container;
```

Comment:
Maybe we should also make `base` variable locally scoped for each test. What do you think?

Reply:
Thanks, sounds good to me 👍 Done in 87ac0e41381064fc3b91e91bc1c3174e65c49ee5